spras/config/schema.py

```diff
@@ -121,12 +121,14 @@ class Dataset(BaseModel):
     # validation & coercion logic before we check it against our own
     # requirements
     label: Annotated[str, AfterValidator(label_validator("Dataset"))]
+    category: Optional[str] = None
+    "The dataset category, for working with multiple datasets at once in the configuration."
```

Comment:
I would be more specific on this: 

```suggestion
    "The dataset category, for working with multiple datasets collections at once in the configuration."
```